### ISSUE TITLE
<sematics> element in MathML is not found in caniuse.com

### ISSUE THREAD

**Issue by kyroluskamal** [Fyrd/caniuse#5083]
I found the element <sematics> in MDN web docs (https://developer.mozilla.org/en-US/docs/Web/MathML/Element/semantics) and when I searched for it in caniuse.com, I found 0 search results. 

So, please try to add it.

**Comment by Fyrd** [comment]
Yeah I figure since MathML itself isn't terribly popular or well supported across browsers it didn't seem worth including MDN's MathML data beyond basic support at https://caniuse.com/#feat=mathml. Will leave this issue open in case further interest arises.

**Comment by Schweinepriester** [comment]
@Fyrd 

<https://developer.mozilla.org/en-US/docs/Web/MathML/Element/semantics#browser_compatibility> has a support entry, but it doesn't appear? Neither via https://caniuse.com/?search=semantics nor <https://caniuse.com/?search=mathml>…?
